instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/sessionreplay/SrTest.kt

```diff
@@ -54,13 +88,43 @@ internal abstract class SrTest<R : Activity, T : MockServerActivityTestRule<R>>
         }
     }
 
+    private fun updateExpectedDataInPayload(
+        handledRequests: List<HandledRequest>,
+        expectedPayloadFileName: String
+    ) {
+        val records = handledRequests
+            .mapNotNull { it.extractSrSegmentAsJson()?.asJsonObject }
+            .flatMap { it.getAsJsonArray("records") }
+            .filter { it.asJsonObject.get("type").asString != "11" }
+            .map { it.dropInconsistentProperties() }
+        assertThat(records.size).isGreaterThan(2)
```

Comment:
also here, why 2?

Reply:
because I am taking `3` bellow so want to make sure I have where to take from...I could actually just take 3 and have the exception in case it doesn't have data yet but I rather like it better this way, more readable